apps/web/pages/auth/logout.tsx

```diff
@@ -1,6 +1,6 @@
 import type { GetServerSidePropsContext } from "next";
-import { signOut, useSession } from "next-auth/react";
-import { useRouter } from "next/navigation";
+import { destroy, useSession } from "next-auth/react";
```

Comment:
As with our current version we get this error:

Module '"next-auth/react"' has no exported member 'destroy'.